modules/log6x-collection-setup.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-collection-setup_{context}"]
+= Setting up log collection
+
+This release of Cluster Logging requires administrators to explicitly grant log collection permissions to the service account associated with *ClusterLogForwarder*. This was not required in previous releases for the legacy logging scenario consisting of a *ClusterLogging* and, optionally, a *ClusterLogForwarder.logging.openshift.io* resource.
+
+The {clo} provides `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles, which enable the collector to collect audit logs, application logs, and infrastructure logs respectively.
+
+Setup log collection by binding the required cluster roles to your service account.
+
+== Legacy service accounts
+To use the existing legacy service account `logcollector`, create the following *ClusterRoleBinding*:
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-application-logs system:serviceaccount:openshift-logging:logcollector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs system:serviceaccount:openshift-logging:logcollector
+----
+
+Additionally, create the following *ClusterRoleBinding* if collecting audit logs:
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-audit-logs system:serviceaccount:openshift-logging:logcollector
+----
+
+
+== Creating service accounts
+.Prerequisites
+
+* The {clo} is installed in the `openshift-logging` namespace.
+* You have administrator permissions.
+
+.Procedure
+
+. Create a service account for the collector. If you want to write logs to storage that requires a token for authentication, you must include a token in the service account.
+
+. Bind the appropriate cluster roles to the service account:
++
+.Example binding command
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user <cluster_role_name> system:serviceaccount:<namespace_name>:<service_account_name>
+----
+
+=== Cluster Role Binding for your Service Account
+The role_binding.yaml file binds the ClusterLogging operator's ClusterRole to a specific ServiceAccount, allowing it to manage Kubernetes resources cluster-wide.
+
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding
+roleRef:                                           <1>
+  apiGroup: rbac.authorization.k8s.io              <2>
+  kind: ClusterRole                                <3>
+  name: cluster-logging-operator                   <4>
+subjects:                                          <5>
+  - kind: ServiceAccount                           <6>
+    name: cluster-logging-operator                 <7>
```

Comment:
I don't know what this example should do. I would assume we already have a generic example of "what is a ClusterRoleBinding" somewhere in the OpenShift docs. If this is an example for "binding one of the provided ClusterRoles to the collector ServiceAccount" then it should not reference the operator. For example:

```suggestion
  name: logcollector-collect-application
roleRef:                                           <1>
  apiGroup: rbac.authorization.k8s.io              <2>
  kind: ClusterRole                                <3>
  name: collect-application-logs                   <4>
subjects:                                          <5>
  - kind: ServiceAccount                           <6>
    name: logcollector                 <7>
```